doc/nrf/releases/release-notes-2.4.4.rst

```diff
@@ -0,0 +1,101 @@
+.. _ncs_release_notes_244:
+
+|NCS| v2.4.4 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.4.0 <ncs_release_notes_240>`, :ref:`nRF Connect SDK v2.4.1 <ncs_release_notes_241>`, :ref:`nRF Connect SDK v2.4.2 <ncs_release_notes_242>` and :ref:`nRF Connect SDK v2.4.3 <ncs_release_notes_243>`.
+The changes have an effect on Matter and Bluetooth®.
```

Comment:
I propose adding a line here saying something like 
"This release adds workaround for Anomaly 165 and Anomaly 168 that is affecting the nRF5340 SoC."

Reply:
@bjorn-tore-taraldsen , @shanthanordic please confirm.

Reply:
I'm okay with this